### DISCUSSION TITLE
Can't get into admin or see the demo page

### DISCUSSION THREAD

**Discussion by tomsoel** [pimcore/pimcore#14101] [Q&A / Support]
Hello,
I installed Pimcore in a folder "my-project" on a local Laragon server with the "Override All" parameter in httpd.conf.
I didn't get any error during the installation, however, I get a "page not found" with the url localhost/my-project/admin or localhost/admin.
Despite my research and re-installations (skeleton or demo), I did not find any solution.
I tried on another distant server (OVH) but I get the same issue.
Could you give me some suggestions?
Thanks for your help !

**Comment by kingjia90** [comment]
i don't know Laragon and your current settings, but could be related to Apache.
Please check this page about htaccess https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/Apache_Configuration.html

**Reply by tomsoel** [reply]
Thx a lot for reply
I'll have a look at this

**Reply by kingjia90** [reply]
did that solve the problem?

**Reply by tomsoel** [reply]
Sorry for the delay in responding!
No, unfortunately, despite several attempts, I have encountered this problem
Fortunately, I could test Pimcore using the online demo
I'll do some more tests later
Thank you very much for coming back to me for this topic!
